test/server-unit/payroll_test_unit/titles.spec.js

```diff
@@ -0,0 +1,184 @@
+const { expect } = require('chai');
+const sinon = require('sinon');
+
+// Import the DB module to mock it
+const db = require('../../../server/lib/db');
+
+// Import the controller to test
+const controller = require('../../../server/controllers/admin/titles');
+
+describe('test/server-unit/payroll-test-unit/title_employee', () => {
+
+  let req;
+  let res;
+  let next;
+
+  beforeEach(() => {
+    req = { params : {}, body : {} };
+    res = {
+      status : sinon.stub().returnsThis(),
+      json : sinon.stub(),
+    };
+    next = sinon.spy();
+  });
+
+  afterEach(() => {
+    sinon.restore(); // reset mocks after each test
+  });
+
+  // ---------------------------------------------------------
+  it('list() should return status 200 with the list of job titles', async () => {
```

Comment:
It looks like this test is testing expressjs rather than BHIMA.  I'm not sure these kinds of tests are valuable.

Correct me if I am wrong, but this (1) fakes the DB connection, (3) calls controller.list() and then checks if the fake data was returned as JSON.  This is basically what our integration tests do, and I'm not sure what this adds over the integration tests.